src/scripts/components/showcase-selector/showcase-selector.tsx

```diff
@@ -36,12 +36,19 @@ const ShowcaseSelector: FunctionComponent = () => {
           id="storiesSelected"
           values={{numberSelected: selectedIds.length}}
         />
-        <Button
-          className={styles.button}
-          label={'play'}
-          link={`/showcase/${selectedIds.join('&')}/0/title`}
-          icon={PlayIcon}
-        />
+        {selectedIds.length === 0 ? (
+          <Button
```

Comment:
The button should have a "disabled" property which sets the real disabled attribute on the button element (if there is one) and handles the styling automatically

Reply:
Yeah i thought so too but in the button component we either have a button or a link. And in this case it's a link.

Reply:
yes, but the cool thing about this component is that you shouldn't care when using it. so when you set `disabled` the button should get the disabled attribute and be styles correctly and a link tag should prevent the ink from being executed (preventDefault) and also be styled like it is disabled